src/pages/home/HeaderView.js

```diff
@@ -25,65 +29,85 @@ const propTypes = {
+                {props.isSmallScreenWidth && (
                     <Pressable
-                        onPress={() => {
-                            const {participants} = props.report;
-                            if (participants.length === 1) {
-                                Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
-                            }
-                        }}
+                        onPress={props.onNavigationMenuButtonClicked}
+                        style={[styles.LHNToggle]}
                     >
-                        <MultipleAvatars avatarImageURLs={props.report.icons} />
+                        <Icon src={BackArrow} />
                     </Pressable>
-                    <Header title={props.report.reportName} />
-                    <View style={[styles.reportOptions, styles.flexRow]}>
+                )}
+                {props.report && props.report.reportName ? (
+                    <View
+                        style={[
+                            styles.flex1,
+                            styles.flexRow,
+                            styles.alignItemsCenter,
+                            styles.justifyContentBetween,
+                        ]}
+                    >
                         <Pressable
-                            onPress={() => togglePinnedState(props.report)}
-                            style={[styles.touchableButtonImage, styles.mr0]}
+                            onPress={() => {
+                                if (participants.length === 1) {
+                                    Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
+                                }
+                            }}
                         >
-                            <Icon src={Pin} fill={props.report.isPinned ? themeColors.heading : themeColors.icon} />
+                            <MultipleAvatars avatarImageURLs={props.report.icons} />
                         </Pressable>
+                        <View style={[styles.flex1, styles.flexRow]}>
+                            <OptionRowTitle
+                                option={reportOption}
+                                tooltipEnabled
+                                numberOfLines={2}
+                                style={[styles.headerText]}
+                            />
+                        </View>
+                        <View style={[styles.reportOptions, styles.flexRow]}>
+                            <Pressable
+                                onPress={() => togglePinnedState(props.report)}
+                                style={[styles.touchableButtonImage, styles.mr0]}
+                            >
+                                <Icon src={Pin} fill={props.report.isPinned ? themeColors.heading : themeColors.icon} />
+                            </Pressable>
+                        </View>
                     </View>
-                </View>
-            ) : null}
+                ) : null}
```

Comment:
Same here... I know that this was like this before, but we can probably simplify this ternary to just use `??` instead